spec/integration/rack_spec.rb

```diff
@@ -105,6 +105,14 @@
 
       specify { expect(last_response.status).to eql 200 }
       specify { expect(last_response.body).to eq batch_compiled_json }
+      specify {
+        expect(last_response.headers).to eq({
+          "Content-Type"   => "application/json",
+          "ETag"           => "44eaafebcdd94ddac4c4f07ecf0cc80f",
+          "Last-Modified"  => nil,
```

Comment:
@dazoakley, I wasn't sure about this test. There wasn't a test which checked headers before, but with this, I have the Last-Modified as nil. I didn't want to have to parse the Dates again from the components, as I'm doing that in Alephant-Storage. It just seems odd that it's nil here.